tests/test_forcing.py

```diff
@@ -0,0 +1,1025 @@
+from esmvalcore.experimental import get_recipe
+import deepdiff
+import copy
+import pytest
+from ewatercycle.forcing import update_marrmot, update_lisflood, update_hype, update_wflow, update_pcrglobwb
+
+TEST_INPUT_MARRMOT = (
+    {},
+    {
+        'startyear': 1990,
+        'endyear': 2018,
+        'forcings': ['ERA-Interim', 'ERA5'],
+        'shapefile': 'Meuse/Meuse.shp',
+    },
+    {
+        'startyear': 1234
+    },
+    {
+        'endyear': 4321
+    },
+    {
+        'forcings': ['ERA5']
+    },
+    {
+        'shapefile': 'Rhine/Rhine.shp'
+    },
+)
+
+TEST_INPUT_LISFLOOD = (
+    {},
+    {
+        'startyear': 1990,
+        'endyear': 1990,
+        'forcings': ['ERA-Interim', 'ERA5'],
+        'shapefile': 'Lorentz_Basin_Shapefiles/Meuse/Meuse.shp',
```

Comment:
Same, where does this path lead to?

Reply:
This is also the default value in the `esmvaltool` recipe.